gcloud/monitoring/client.py

```diff
@@ -195,6 +203,121 @@ def metric_descriptor(self, type_,
             display_name=display_name,
         )
 
+    @staticmethod
+    def metric(type_, labels):
+        """Construct a metric object.
+
+        The type of the Metric should match the type of a MetricDescriptor.
+        For a list of built-in Metric types and the associated labels, see:
+
+        https://cloud.google.com/monitoring/api/metrics
+
+        Here is an example of creating using a built-in metric type:
+
+             >>> metric = client.metric('instance/cpu/utilization' labels={
+             ...    'instance_name': 'my-instance',
+             ... })
+
+        For custom metrics, the simplest way to get the type is from the
+        :class:`gcloud.monitoring.metric.MetricDescriptor` class used to
```

Comment:
I personally didn't realize the tilde existed until @rimey used it 1+ year into the project. It's the preferred because usually we just want a link and the name of the relevant object. Sometimes the entire qualified import path is desired, it really depends on the author.